deepchem/molnet/load_function/tests/test_swissprot_loader.py

```diff
@@ -0,0 +1,23 @@
+"""
+Tests for swissprot loader.
+"""
+
+import os
+import numpy as np
+from deepchem.molnet import load_swissprot
+def test_swissprot_loader():
```

Comment:
Can you add a `@slow` annotation to this function?

Reply:
Okay !

Reply:
Can you make sure to run `yapf` on the files? I think the formatting might have some issues